conceptnet5/vectors/__init__.py

```diff
@@ -1,13 +1,15 @@
-from conceptnet5.nodes import standardized_concept_uri, uri_to_label
-from conceptnet5.language.lemmatize import lemmatize_uri
-from sklearn.preprocessing import normalize
 import re
-import pandas as pd
+
 import numpy as np
+import pandas as pd
+from sklearn.preprocessing import normalize
 
+from conceptnet5.languages import ALL_LANGUAGES
+from conceptnet5.nodes import standardized_concept_uri, uri_to_label
 
 DOUBLE_DIGIT_RE = re.compile(r'[0-9][0-9]')
 DIGIT_RE = re.compile(r'[0-9]')
+CONCEPT_RE = re.compile(r'/c/({})/.+'.format('|'.join(ALL_LANGUAGES)))
```

Comment:
The main case I'm worried about is, what happens when we get data in a language that's not in `ALL_LANGUAGES`? Will it get filtered out at the appropriate time in the build process, or will it look like an unreasonable English word?

I want to make sure that there's a sensible way to collect more data and possibly extend the `ALL_LANGUAGES` list.